### DISCUSSION TITLE
FundMeTest.t.sol error

### DISCUSSION THREAD

**Discussion by GTK-ARJUN** [Cyfrin/foundry-full-course-cu#983] [Q&A]
I am getting an error as  below after following the YouTube [video](https://youtu.be/sas02qSFZ74?si=_DE5dCo4PP2uxOUo&t=1284):

`Error (6160): Wrong argument count for function call: 0 arguments given but expected 1.
  --> test/FundMeTest.t.sol:11:18:
   |
11 |         fundme = new FundMe();
   |                  ^^^^^^^^^^^^
`

![fundmetest t sol](https://github.com/Cyfrin/foundry-full-course-f23/assets/58646244/ddfb2092-b953-48e2-8dd8-09dde22a0a0c)

**Comment by theirrationalone** [comment]
You might expecting an Argument into your FundMe Contract's `constructor` special function. Please verify & make sure you're not expecting or passing any argument into your FundMe Contract's `constructor` function.

**Comment by auroralzone** [comment]
https://github.com/Cyfrin/foundry-full-course-f23/discussions/81#discussioncomment-6200898
